bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -598,6 +550,56 @@ class PresentationUploader extends Component {
     });
   }
 
+  renderToastItem(item) {
+    const isUploading = !item.upload.done && item.upload.progress > 0;
+    const isConverting = !item.conversion.done && item.upload.done;
+    const hasError = item.conversion.error || item.upload.error;
+    const isProcessing = (isUploading || isConverting) && !hasError;
+
+    const {
+      intl, selectedToBeNextCurrent,
+    } = this.props;
+
+    const itemClassName = {
+      [styles.done]: !isProcessing && !hasError,
+      [styles.err]: hasError,
+      [styles.loading]: isProcessing,
+    };
+
+    const statusInfoStyle = {
+      [styles.textErr]: hasError,
+      [styles.textInfo]: !hasError,
+    };
+
+    let icon = isProcessing ? 'blank' : 'check';
+    if (hasError) icon = 'circle_close';
+
+    return (
+      <div
+        key={item.id}
+        className={styles.uploadRow}
+        onClick={() => {
+          if (hasError || isProcessing) Session.set('showUploadPresentationView', true);
+        }}
```

Comment:
Would be best practice to create handler function instead of inline, inline should be one liner max we can use ternary operator, In this case 

```
onClick={handleToastItemClick}
```

add `handleToastItemClick` above return

```
const handleToastItemClick = () => {
  if (hasError || isProcessing) {
    Session.set('showUploadPresentationView', true);
  }
}
```

Reply:
fixed